gems/quilt_rails/lib/generators/quilt_rails/quilt/rails_setup/rails_setup_generator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module QuiltRails
+  module Generators
+    module Quilt
+      class RailsSetupGenerator < Rails::Generators::Base
```

Comment:
:TIL: The API's being used here are really cool. Can you link the docs for where `say`, `route`, and `copy_file` come from?

Reply:
😅 The doc was a combination of 
- what exist in `quilt_rails` already
- https://github.com/rails/rails/blob/master/guides/source/generators.md
- https://guides.rubyonrails.org/generators.html#adding-command-line-arguments
- and some patterns I see in https://github.com/Shopify/shopify_app/tree/master/lib/generators/shopify_app